attestation-agent/attester/src/snp/mod.rs

```diff
@@ -44,6 +44,11 @@ impl Attester for SnpAttester {
             .get_ext_report(None, Some(data), Some(0))
             .context("Failed to get attestation report")?;
 
+        let certs = match certs {
+            Some(c) => c,
+            None => bail!("Certificate chain was not provided"),
+        };
```

Comment:
I think we are going to add support in Trustee for the verifier to pull the cert chain if needed, so I would prefer not to have a fatal error here.

Reply:
I've removed the bail and just unwrapped the value.

Reply:
Unwrapping it is still fatal. If there is no cert chain we probably want to pass an empty Vec. You could do `cert_chain: certs.unwrap_or(vec![])` or we could change the SnpEvidence struct to have `cert_chain: Option<Vec<CertTableEntry>>`

Reply:
Looks like the SEV crate returns `Option<Vec<CertTableEntry>>` so you might just change the struct and use that value directly.